src/main/java/qi/ui/Ui.java

```diff
@@ -0,0 +1,141 @@
+package qi.ui;
+
+import qi.task.Task;
+import qi.tasklist.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * Represents the interactions of the chatbot with users.
+ */
+public class Ui {
```

Comment:
I like the consistent naming of the methods 👍

Reply:
Thank you 🙇